chopper.py

```diff
@@ -13,6 +13,14 @@
 from typing import Any
 
 
+## [CG] Putting this up here since there's not really 1 place to put it: I
+##      might spend some time using things like filepath.open and
+##      filepath.rename instead of open and os.*. Admittedly there's not a ton
+##      of value that comes from doing this, it's just a little more consistent
+##      with using pathlib.Path everywhere you can.
+
+
+## [CG] Oooh fahncy, love it
```

Comment:
Easier than all the `self.key = key` bs, but I actually ended up dropping the class in favor of just passing around the args Namespace object.